reference/api/rest.rst

```diff
@@ -0,0 +1,358 @@
+| Erase a product value        |.. code-block:: json                  |.. code-block:: json                 |.. code-block:: json                 |
+|                              |                                      |                                     |                                     |
+|                              |  {                                   |  {                                  |  {                                  |
+|                              |      "values": {                     |      "values": {                    |      "values": {                    |
+|                              |          "sku": [                    |          "name": [                  |          "sku": [                   |
+|                              |              {                       |              {                      |              {                      |
+|                              |                  "locale": null,     |                  "locale": "en_US", |                 "locale": null,     |
+|                              |                  "scope": null,      |                  "scope": null,     |                 "scope": null,      |
+|                              |                  "data": "mug"       |                  "data": null       |                 "data": "mug"       |
+|                              |              }                       |              }                      |              }                      |
+|                              |          ],                          |          ]                          |          ],                         |
+|                              |          "name": [                   |      }                              |          "name": [                  |
+|                              |             {                        |  }                                  |              {                      |
+|                              |                  "locale": "en_US",  |                                     |                  "locale": "en_US", |
+|                              |                  "scope": null,      |                                     |                  "scope": null,     |
+|                              |                  "data": "Mug"       |                                     |                  "data": null       |
+|                              |             }                        |                                     |              }                      |
+|                              |          ]                           |                                     |          ]                          |
+|                              |      }                               |                                     |      }                              |
+|                              | }                                    |                                     |  }                                  |
```

Comment:
Removing a product value should have for effect to remove the product value in the resource.